cmd/kueue/main.go

```diff
@@ -99,12 +102,22 @@ func init() {
 }
 
 func main() {
+	var metricsAddr string
+	var secureMetrics bool
+	var enableHTTP2 bool
+	var tlsOpts []func(*tls.Config)
+	var featureGates string
 	var configFile string
+
+	flag.StringVar(&metricsAddr, "metrics-bind-address", "0", "The address the metrics endpoint binds to. "+
```

Comment:
metricsAddr is already available in the kueue configuration.

Reply:
Oh, I see, thanks!